airbyte-webapp/src/packages/cloud/services/cloudApi/useSpeakeasyRedirect.ts

```diff
@@ -2,10 +2,13 @@ import { getSpeakeasyCallbackUrl } from "packages/cloud/lib/domain/cloudApi";
 import { useSuspenseQuery } from "services/connector/useSuspenseQuery";
 import { useDefaultRequestMiddlewares } from "services/useDefaultRequestMiddlewares";
 
+import { useConfig } from "../config";
+
 const SPEAKEASY_QUERY_KEY = "speakeasy-redirect";
 
 export const useSpeakeasyRedirect = () => {
-  const config = { apiUrl: "/cloud_api" };
+  const { cloudApiUrl } = useConfig();
+  const config = { apiUrl: cloudApiUrl };
```

Comment:
this config is incorrectly pointing the to `/cloud` prefix, instead of the intended `/cloud_api`